stdlib/gc.mli

```diff
@@ -419,7 +419,10 @@ external eventlog_pause : unit -> unit = "caml_eventlog_pause"
    Traces are collected if the program is linked to the instrumented runtime
    and started with the environment variable OCAML_EVENTLOG_ENABLED.
    Events are flushed to disk after pausing, and no new events will be
-   recorded until [eventlog_resume] is called. *)
+   recorded until [eventlog_resume] is called.
+
+   @since 5.0
```

Comment:
Oh dear, there was so much back-and-forth over the format of the version nuymber in #11212 that no one noticed that the version itself is wrong!
```suggestion
   @since 4.11
```

this also wants changing below on Line 436!